### ISSUE TITLE
Intermediate.Conformance.1002 - Entities with properties that have null values

### ISSUE THREAD

**Issue by uythoang** [RESOStandards/RESO-WebAPI-Client-dotNet#13]
### Description

I'm having some issues with the `$search` verification of this test. To my understanding, this is how that part of this test works:

1. From the metadata doc, it selects the first EntitySet in the EntityContainer. In our case, this is `Member`.
2. It then selects the first Member property of type Edm.String, which in our case is `JobTitle`.
3. Calling `/api/reso/odata/Member`, it gets back a list of members from which it selects the first entity and looks up that member's JobTitle. In this case that value happens to be `JobTitle: null`.
4. Due to the null, it then attempts to call `/api/reso/odata/Member?$search=` which results in an error.

Aside from making sure that every Member has a JobTitle (which is not possible because that property is nullable), how can we get $search to work here?

**Comment by sschuessler** [comment]
I agree and this is a known issue.  The null if it comes back should be text qualified or it should go through the data until it finds a value.  I will look at fixing that on the next release.
